utils/evaluator.py

```diff
@@ -34,17 +39,40 @@ def __call__(self, data_loader):
             self.current_intersection = torch.sum(prediction * ground_truth)
             self.total_pixels = torch.sum(ground_truth) + torch.sum(prediction)
             self.current_union = self.total_pixels - self.current_intersection
-            total_iou += self.iou()
-            total_f_score += self.f_score()
-        return total_iou / len(data_loader), total_f_score / len(data_loader)
+            total_iou += self.__iou()
+            total_f_score += self.__f_score()
+        self.mean_iou = total_iou / len(data_loader)
+        self.mean_f_score = total_f_score / len(data_loader)
+        self.dataset_name = type(data_loader.dataset).__name__
+        self.add_row_to_table()
 
-    def iou(self):
+    def __iou(self):
         return self.current_intersection / \
                ((self.current_union - self.current_intersection) + self.epsilon)
 
-    def f_score(self):
+    def __f_score(self):
         return 2 * self.current_intersection / (self.total_pixels + self.epsilon)
 
+    def add_row_to_table(self):
+        self.table.add_rows([["", self.mean_iou, self.mean_f_score]])
+        print(self.table.draw())
+
+    def print_metrics(self):
+        print(self.table.draw())
```

Comment:
There is no clear usage for these functions, they can be part of `__call__` and be printed after the evaluation process ends.